tests/functional/c/consider/consider_using_f_string.py

```
@@ -0,0 +1,107 @@
+    "String {Param_1}".format(Param_1=PARAM_1)  # [consider-using-f-string]
+    "{} {}".format(PARAM_1, PARAM_2)  # [consider-using-f-string]
+    "{Par_1}{Par_2}".format(Par_1=PARAM_1, Par_2=PARAM_2)  # [consider-using-f-string]
+    "{Param_1}".format(*PARAM_LIST_SINGLE)  # [consider-using-f-string]
+    "{Param_1}".format(**PARAM_DICT_SINGLE)  # [consider-using-f-string]
+    "String %s" % (PARAM_1)  # [consider-using-f-string]
+    "String %s %s" % (PARAM_1, PARAM_2)  # [consider-using-f-string]
+    "String %s" % (PARAM_LIST_SINGLE)  # [consider-using-f-string]
+
+
+def assignment_good():
+    A = "String {}, {} or {}".format(*PARAM_LIST)
```

Ruff (0.11.9)

Local variable `A` is assigned to but never used

Remove assignment to unused variable `A`

(F841)